src/engraving/dom/hairpin.cpp

```diff
@@ -544,4 +534,30 @@ String Hairpin::accessibleInfo() const
     }
     return rez;
 }
+
+//---------------------------------------------------------
+//   translatedSubtypeUserName
+//---------------------------------------------------------
+
+String Hairpin::translatedSubtypeUserName() const
+{
+    switch (hairpinType()) {
+    case HairpinType::CRESC_HAIRPIN:
+        return mtrc("engraving", "Crescendo");
+    case HairpinType::DECRESC_HAIRPIN:
+        return mtrc("engraving", "Decrescendo");
+    default:
+        return mtrc("engraving", "Custom");
```

Comment:
Shouldn't `CRESC_LINE` and `DECRESC_LINE` be handled here too?

Reply:
They're not handled above in `accessibleInfo` either, should it be changed there too?

Reply:
Yes, good point, that would make sense. I wouldn't know why it would currently say "Custom"…